_posts/2015-12-13-weak-tests-hurt.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title:  "Weak Tests Hurt"
+date:   2015-11-15 10:00:00
+categories: git development
+author: kent.safranski
+cover: /assets/images/covers/failed-test.jpg
```

Comment:
I haven't tried it for cover images yet, but have [had luck](https://github.com/jhabdas/jhabdas.github.io/blob/a0f1ec8e4460a7c10137e9ef429b250be017ab48/plugins/aws_s3_imagetag.rb) with the `aws_s3_imagetag.rb` plugin to push images to AWS. As we continue to add static assets we will see the app `clone`/`push` process slow and want to consider upload to S3 around that time.